src/Roslyn.Diagnostics.Analyzers/Core/RoslynDiagnosticIds.cs

```diff
@@ -54,8 +54,8 @@ internal static class RoslynDiagnosticIds
         public const string PublicApiFileMissing = "RS0048";
         public const string TemporaryArrayAsRefRuleId = "RS0049";
 
-        public const string WrapStatementsRuleId = "RS0100";
-        public const string BlankLinesRuleId = "RS0101";
-        public const string BracePlacementRuleId = "RS0102";
+        //public const string WrapStatementsRuleId = "RS0100"; // Now ported to dotnet/roslyn https://github.com/dotnet/roslyn/pull/50358
+        //public const string BlankLinesRuleId = "RS0101"; // Now ported to dotnet/roslyn https://github.com/dotnet/roslyn/pull/50358
+        //public const string BracePlacementRuleId = "RS0102"; // Now ported to dotnet/roslyn https://github.com/dotnet/roslyn/pull/50358
```

Comment:
I assume these IDs shouldn't be re-used.

If I'm wrong, then we can entirely delete it instead of commenting it.

Reply:
Fine with just deleting these.

Reply:
No, this was definitely correct.